1st module Capstone

Sirvelasque opened the pull request:
- Create 2 pages of the website About and home pages
- Headline seccion with custom background
- Responsive design
- Features section made with javascript
- Custom topic

# link to loom video
- https://www.loom.com/share/dcd10db783484b3395cbb1cac4e03c63

Sirvelasque committed:
Create navBar, set color variables and normalice

created the html for the navBar adding icons, set colors in variables
create a css with rules to normalice the page

Sirvelasque committed:
add links to navbar items

Sirvelasque committed:
set mobile nav menu

Sirvelasque committed:
finish mobile nav menu, set nav button

Sirvelasque committed:
create headline section

Sirvelasque committed:
fix text color and margin, change images

the image adds and changes are done to because they were to heavy

Sirvelasque committed:
set header background by image and icons

Sirvelasque committed:
set program section and items

Sirvelasque committed:
set program icons and text, fix gap

background icon containers had a big width that created a gap, fixed
setted program icons, title and text

Sirvelasque committed:
set program items layout

gap, width and height of the inside items such as the icons, title and text

Sirvelasque committed:
set font family and program section btn

Sirvelasque committed:
resize and colors

set button color and sizes

Sirvelasque committed:
add features seccion and footer

Sirvelasque committed:
set source of sponsor images

Sirvelasque committed:
set partner logos size and place

Sirvelasque committed:
finish footer

add logo with text and copyright note

Sirvelasque committed:
sett bars icon interaction

Sirvelasque committed:
change font size and color on partner h3

Sirvelasque committed:
add image files and create JS objects

the javascript object contain the speakers info

Sirvelasque committed:
implement feature section

add title and elements by innethtml looping a template

Sirvelasque committed:
adjust desktop navbar

Sirvelasque committed:
set desktop headline and program items container

Sirvelasque committed:
adjust desktop footer and speakers min width

Sirvelasque committed:
initialice about page

Sirvelasque committed:
COPY AND AJUST NAVBAR

Sirvelasque committed:
copu and ajust index background to about

Sirvelasque committed:
create and justify headline text

Sirvelasque committed:
set and justify content on logo and past submmit sections

Sirvelasque committed:
adjust past submmits images

Sirvelasque committed:
copy and fix footer from index to about

Sirvelasque committed:
justify the content for desktop version

Sirvelasque committed:
fix gap

Sirvelasque committed:
fix about link in about page

Sirvelasque committed:
Update README.md

Sirvelasque committed:
linter erros

Sirvelasque committed:
Merge branch 'Mobile_version' of https://github.com/Sirvelasque/capstonePage into Mobile_version

Sirvelasque committed:
fix linter erros

Sirvelasque committed:
fix linter errors

Sirvelasque committed:
linter erros

Sirvelasque committed:
eslint error fix

Sirvelasque committed:
Update app.js

micheaol commented on html/about.html:
```diff
@@ -1 +1,137 @@
+        </div>
+        <div class="footerItem">2022 Trampoline Dev Honduras <br/> some Rights Reserved</div>
+      </div>
+    </footer>
```
Please ensure that the `About Page footer` has a dark background according to the requirement.

micheaol commented on index.html:
```diff
@@ -3,14 +3,165 @@
+      <button id="Program-btn">Join TD Global Summit 2022</button>
+      <div class="seeprogram"><a href="#">Se the whole program</a></div>
+    </section>
+    <!--feature spekear-->
+    <section id="features">
+    </section>
```
Please ensure that the pages look almost identical to the original design.

Your Implementation | Figma Design
:---:|:---:
   
![your_design](https://user-images.githubusercontent.com/10332499/153350762-636e2713-e5d7-4014-be1e-05a43c5ec1ca.PNG) | ![design](https://user-images.githubusercontent.com/10332499/153350789-b65fb35c-78a8-445d-b02e-305b066e722d.PNG)

micheaol reviewed (changes requested):
Hi @Sirvelasque , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️

- _Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Sirvelasque committed:
add squares pattern background to speakers

added a background by css that changes on the breakpoint with a similar one but with another grayscale
--also fix right gap bug

Sirvelasque committed:
responsive footer adjustments

- made the partners section not display on the desktop version
- the bottom part becomes "dark-mode" on desktop version by change the logo by css background

Sirvelasque committed:
Update main768.css

Deepakdanger reviewed (approved):
Hi @Sirvelasque, 

### Status: Approved ✔️

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
